react-graphql-apollo-tutorial.md

```diff
@@ -491,16 +491,16 @@ ReactDOM.render(
 );
 {{< /highlight >}}
 
-That's it. Now you have implicit access to the Apollo Client in your React view-layer. It says implicit, because most often you will not use the client in an explicit way. You will see in the next section what this means.
+ Now you have implicit access to the Apollo Client in your React view-layer. It says implicit because most often you will not use the client explicitly. You will see in the next section what this means.
 
 ### Exercises:
 
-* read more about {{% a_blank "configuring and connecting Apollo Client to React" "https://www.apollographql.com/docs/react/essentials/get-started.html" %}}
-* invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5FHMHW8" %}}
+* Read more about {{% a_blank "configuring and connecting Apollo Client to React" "https://www.apollographql.com/docs/react/essentials/get-started.html" %}}
+* Invest 3 minutes of your time and take the {{% a_blank "quiz" "https://www.surveymonkey.com/r/5FHMHW8" %}}
 
 {{% chapter_header "GraphQL Query with Apollo Client in React" "react-apollo-client-query" %}}
 
-In this section, you are going to implement your first GraphQL query using Apollo Client in React. In previous sections, you have already seen how different entities, such as the current user (viewer) or repositories, can be queried from GitHub's GraphQL API. This time you will do it in React. A Profile component might be the best place to render the current user and the associated repositories of this user. You can start out by using the not yet implemented Profile component in your App component. Implementing the Profile component will be the next step. It makes sense to extract the Profile component at this early point in time, because the App component will later be the static frame around the application whereas components such as Navigation and Footer are static and components such as Profile and Organization are dynamically rendered based on routing (URLs).
+In this section, you will implement your first GraphQL query using Apollo Client in React. You've seen how different entities, such as the current user (viewer) or repositories, can be queried from GitHub's GraphQL API. This time you will do it in React. A Profile component might be the best place to render current users and their associated repositories. Start by using the not-yet-implemented Profile component in your App component, which we'll take care of next.  It makes sense to extract the Profile component now, because the App component will be the static frame around the application later. Components like Navigation and Footer are static, and components such as Profile and Organization are dynamically rendered based on routing (URLs).
```

Comment:
"the best place to render current users and their associated repositories" to "the best place to render the current user and the associated repositories of this user"

Reply:
" to render the current user and its associated repositories"  
"of this user" is redundant, because you attached it already with the word "associated"

Reply:
Agreed.